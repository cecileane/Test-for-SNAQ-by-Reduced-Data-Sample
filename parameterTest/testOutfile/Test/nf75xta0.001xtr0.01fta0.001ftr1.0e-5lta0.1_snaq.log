optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 0.1.
rootname for files: nf75xta0.001xtr0.01fta0.001ftr1.0e-5lta0.1_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 10293
seed: 10293 for run 1
Sat Oct  1 17:39:09 2016

 BEGIN SNaQ for run 1, seed 10293 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 100 (failures: 75) with -loglik=126.74251 and ht_min=[0.2856,1.51213,10.0,0.48942,0.51916]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 29		 29		 2	 |	 0		 0		 --
mvtarget	 21		 21		 0	 |	 0		 0		 --
chdir		 10		 10		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 39		 36		 1	 |	 3		 3		 --
Total		 100		 97		 5	 |	 3		 3		 3
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981974969
((2,(1)#H7:::0.7143969620103587):1.5121335632591373,(3,(5,(6,#H7:::0.28560303798964126):10.0):0.5191607961568443):0.4894188407437991,4);
---------------------
seed: 44888 for run 2
Sat Oct  1 17:39:17 2016

 BEGIN SNaQ for run 2, seed 44888 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 128 (failures: 75) with -loglik=155.49626 and ht_min=[0.33535,0.12573,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 32		 32		 4	 |	 0		 0		 --
mvtarget	 35		 35		 1	 |	 0		 0		 --
chdir		 14		 14		 2	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 45		 31		 1	 |	 8		 8		 --
Total		 128		 114		 9	 |	 9		 9		 5
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 2, -loglik of best 155.49625734554343
(((1,(2)#H7:::0.6269752492614136):2.155859413254681,(4,#H7:::0.3730247507385864):2.155859413254681):0.12573340449611667,3,(5,6):0.335347362611805);
---------------------

Sat Oct  1 17:39:23 2016
MaxNet is ((2,(1)#H7:::0.7143969620103587):1.5121335632591373,(3,(5,(6,#H7:::0.28560303798964126):10.0):0.5191607961568443):0.4894188407437991,4); 
with -loglik 126.74250981974969
