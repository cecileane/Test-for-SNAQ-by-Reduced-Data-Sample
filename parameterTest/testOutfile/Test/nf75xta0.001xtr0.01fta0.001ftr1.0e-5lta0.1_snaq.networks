((2,(1)#H7:::0.7143969620103587):1.5121335632591373,(3,(5,(6,#H7:::0.28560303798964126):10.0):0.5191607961568443):0.4894188407437991,4);, with -loglik 126.74250981974969 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((2,(1,#H7:::0.31010662756792584):3.9953570898644752):0.6696590849307966,(3,(5,(6)#H7:::0.6898933724320742):1.730959479064222):0.4680296681261353,4);, with -loglik 128.27691749543712
((4)#H7:::0.6030197456624,3,(5,(6,(1,(2,#H7:::0.3969802543376):2.457067453309386):1.1867434257536051):0.21737632218831093):0.03078111835089783);, with -loglik 172.62767017834136
(#H7:::0.42615968124301906,(3,(5,(6,(1,(2)#H7:::0.5738403187569809):4.392948970888646):0.16060213701870993):0.03958352749838703):1.6540071459182553,4);, with -loglik 192.34089927965422
((2,(1,((#H7:::0.4025771152272276,5):9.98961809726882,6):0.999977402767896):0.09258847777595229):0.07567776507040637,(3)#H7:::0.5974228847727724,4);, with -loglik 215.12525611451
((2,(1,(#H7:::0.4968715196249039,6):5.669383428304635):0.047206477930871925):0.2043216163137238,(3,(5)#H7:::0.5031284803750962):1.1876850031986386,4);, with -loglik 231.14138163297622
