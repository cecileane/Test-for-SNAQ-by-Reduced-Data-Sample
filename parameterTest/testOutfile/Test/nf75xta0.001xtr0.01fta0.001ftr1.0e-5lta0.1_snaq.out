((2,(1)#H7:::0.7143969620103587):1.5121335632591373,(3,(5,(6,#H7:::0.28560303798964126):10.0):0.5191607961568443):0.4894188407437991,4); -Ploglik = 126.74250981974969
 Dendroscope: ((2,(1)#H7):1.5121335632591373,(3,(5,(6,#H7):10.0):0.5191607961568443):0.4894188407437991,4);
 Elapsed time: 14.402963812 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((2,(1)#H7:::0.7143969620103587):1.5121335632591373,(3,(5,(6,#H7:::0.28560303798964126):10.0):0.5191607961568443):0.4894188407437991,4);, with -loglik 126.74250981974969
 (((1,(2)#H7:::0.6269752492614136):2.155859413254681,(4,#H7:::0.3730247507385864):2.155859413254681):0.12573340449611667,3,(5,6):0.335347362611805);, with -loglik 155.49625734554343
-------
